package.json

```diff
@@ -3443,6 +3449,10 @@
 				{
 					"command": "pr.checkoutFromDescription",
 					"when": "chatSessionType == copilot-swe-agent"
+				},
+				{
+					"command": "pr.applyChangesFromDescription",
```

Comment:
Should have an entry in the command palette menu contribution to hide it from the command palette.